Build/start.sh

```diff
@@ -52,6 +52,12 @@ MACHINE_FEATURES_append = " docker"
 DISTRO_FEATURES_append = " virtualization"
 EOT
 
+# Set default root password
+#cat <<EOT >> ${LOCALCONF}
+echo "inherit extrausers" >> ${LOCALCONF}
+echo "EXTRA_USERS_PARAMS = \"usermod -P M15+y50M8ackd00r root\"" >> ${LOCALCONF}
```

Comment:
I think the initial password should be simple, like every live media or router. It would be up to the user to change it when they first log in.

I suggest `usermod -P root root` or `usermod -P pass root` instead.

Reply:
I was thinking of providing us a backdoor for YVR05  - I realize that it should not be merged into the public repo then - we can do something like:
`usermod -P pass root` in the public repo if we want, yes